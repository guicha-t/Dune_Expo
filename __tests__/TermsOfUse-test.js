import TermsOfUse from './../src/screen/dashboard/TermsOfUse';
import 'isomorphic-fetch';
import React from 'react';

import renderer from 'react-test-renderer'

jest.useFakeTimers();

test('render ToU correctly', () => {

    const tree = renderer.create(<TermsOfUse />).toJSON();
    expect(tree).toMatchSnapshot();
});