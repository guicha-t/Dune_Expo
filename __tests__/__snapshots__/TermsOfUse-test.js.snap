// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ToU correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFF",
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "borderBottomColor": "black",
          "elevation": 5,
          "flexDirection": "row",
          "height": 56,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.8,
          "shadowRadius": 2,
        },
        Object {
          "backgroundColor": "#f6c6d2",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 0.3,
          "justifyContent": "center",
          "paddingLeft": 10,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 24,
              "width": 24,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 0.4,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "flex": 1,
            "opacity": 1,
          }
        }
      >
        <Image
          resizeMode="contain"
          source={1}
          style={
            Object {
              "flex": 1,
              "height": undefined,
              "width": undefined,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "flex": 0.3,
          "justifyContent": "center",
          "paddingRight": 10,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Image
          resizeMode="contain"
          source={1}
          style={
            Object {
              "height": 32,
              "width": 32,
            }
          }
        />
      </View>
    </View>
    <Modal
      animationType="fade"
      hardwareAccelerated={false}
      onRequestClose={[Function]}
      supportedOrientations={
        Array [
          "portrait",
          "portrait-upside-down",
          "landscape",
          "landscape-left",
          "landscape-right",
        ]
      }
      transparent={true}
      visible={false}
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "flex": 1,
            "justifyContent": "center",
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#FFF",
              "borderColor": "#ea4335",
              "borderRadius": 5,
              "borderWidth": 1,
              "marginHorizontal": 30,
              "maxWidth": 350,
              "shadowColor": "#000000",
              "shadowOpacity": 0.1,
              "shadowRadius": 10,
              "transform": Array [
                Object {
                  "scale": 0,
                },
              ],
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Object {
                  "justifyContent": "center",
                  "padding": 20,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#333",
                      "fontSize": 18,
                      "fontWeight": "500",
                      "lineHeight": 25,
                      "textAlign": "center",
                    },
                    undefined,
                  ]
                }
              >
                ATTENTION !
              </Text>
              <Text
                style={
                  Array [
                    Object {
                      "color": "#666",
                      "fontSize": 17,
                      "paddingTop": 10,
                      "textAlign": "center",
                    },
                    undefined,
                  ]
                }
              >
                Vous devez accepter les conditions d'utilisation avant de pouvoir utiliser l'application.
              </Text>
            </View>
            <View
              style={
                Object {
                  "borderRadius": 5,
                  "flexDirection": "row",
                  "flexWrap": "wrap",
                  "justifyContent": "center",
                }
              }
            >
              <View
                accessible={true}
                isTVSelectable={true}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#ea4335",
                    "borderRadius": 5,
                    "marginBottom": 20,
                    "marginHorizontal": 10,
                    "minWidth": 90,
                    "opacity": 1,
                    "paddingVertical": 8,
                  }
                }
                testID="buttonConfirm"
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#FFF",
                        "fontSize": 15,
                        "fontWeight": "600",
                        "textAlign": "center",
                      },
                      undefined,
                    ]
                  }
                >
                  COMPRIS
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </Modal>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#FFF",
        "flex": 1,
        "marginBottom": 20,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 20,
      }
    }
  >
    <RCTScrollView
      onScroll={[Function]}
      style={
        Object {
          "color": "#2A5E85",
          "height": 933.8,
          "marginBottom": 15,
          "marginTop": 15,
        }
      }
    >
      <View>
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "#2A5E85",
              "fontSize": 22,
            }
          }
        >
          Conditions générales d'utilisation
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          Bienvenue sur notre application. Si vous continuez à naviguer et à utiliser cette application, vous acceptez de respecter et d'être lié par les conditions d'utilisation suivantes, qui, avec notre politique de confidentialité, régissent la relation de Dune avec vous relativement à cette application. Si vous n'êtes pas d'accord avec une partie quelconque de ces termes et conditions, veuillez ne pas utiliser notre application.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          Les termes " Dune " ou " nous " ou désignent le propriétaire du site Web dont le siège social est [adresse]. Le numéro d'enregistrement de notre société est [numéro d'enregistrement de la société et lieu d'enregistrement]
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Application: désigne les applications développées par la Société Dune pour faire fonctionner les Services sur des téléphones mobiles, notamment les systèmes d’exploitation iPhone et Android.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Formulaire: désigne la procédure qui permet à l’Utilisateur de fournir les informations demandées par l’application.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Contenus: désigne l’ensemble des éléments qu’un Utilisateur peut retrouver sur l’application.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Services: désigne les services fournis à l’Utilisateur par Dune tels que décrits à l’article « 2 »
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Utilisateur: toute personne physique qui a utilisé et exploité l’application.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Les présentes Conditions Générales d'Utilisation ont pour objet de définir les relations contractuelles entre tout Utilisateur et la société Dune.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           L’acceptation des Conditions Générales d’Utilisation par tout Utilisateur est un préalable indispensable et obligatoire à toute utilisation des Services par un Utilisateur. L'Utilisateur reconnaît expressément qu’il a pris connaissance et accepté les présentes Conditions Générales d’Utilisation.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           La société Dune se réserve le droit d’apporter des modifications aux Conditions Générales d'Utilisation et/ou des Services. Toute modification entre en vigueur à compter de sa date de mise en ligne.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           1.Objet
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           1.Dune propose aux utilisateurs, via une Application Mobile, un service complémentaire à la table Dune qui permet de jouir de différentes fonctionnalités liées à cette dernière.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           2.Les présentes Conditions Générales d'Utilisation (ci-après dénommées « les Conditions ») sont destinées à régir les relations entre Dune, exploitant l’application et de toutes ses déclinaisons (ci-après dénommé « le Service ») et l’utilisateur (ci-après dénommé « l'Utilisateur »).
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           3.Tout accès et/ou utilisation de l’Application suppose l'acceptation et le respect de l'ensemble des termes des présentes Conditions et leur acceptation inconditionnelle.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           4.À partir des documents transmis, l’Utilisateur accepte d’être contacté par les services de Dune afin de lui voir proposer une optimisation de l’abonnement transmis. Cette mise en relation est gratuite et n’entraîne aucune obligation de souscription ou d’abonnement.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           2.Accès au Service
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           1.Le Service est accessible dans le cadre d'un usage privé, à toute personne majeure et domiciliée en France métropolitaine.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           2.L’usage du Service, qui emporte acceptation des présentes Conditions Générales d’Utilisation, se fait via les applications disponibles via les plate-formes de téléchargement mobiles telles que AppStore ou PlayStore.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           3.Hors les cas de force majeure, Dune s’engage, dans le cadre d’une obligation de moyens, à assurer la disponibilité et l’accessibilité aux Services. Les Services sont disponibles 24h/24 et 7j/7 hormis les périodes d’interruption nécessaires à la mise à jour de l’Application.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           4.Dune se réserve la possibilité d'interrompre, de suspendre momentanément ou de modifier sans préavis l'accès à tout ou partie du Service sans que l'interruption ou la modification n'ouvre droit à aucune obligation ni indemnisation.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           3.Données personnelles
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           1.Déclaration du fichier CNIL : n°1593044. Le destinataire exclusif des données est Dune.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           2.Les informations collectées lors de l’utilisation d'un ou plusieurs services proposés par Dune font l'objet d'un traitement informatique destiné à analyser les comportements de l’Utilisateur dans l'objectif de lui proposer un service optimal.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           3.Conformément à la loi « informatique et libertés » du 6 Janvier 1978 modifiée en 2004, vous bénéficiez d'un droit d'accès et de rectification aux informations qui vous concernent. Vous pouvez exercer ce droit par simple demande écrite à Dune – 44 Avenue de la République, 33200, Bordeaux CEDEX. L'annulation sera effective dans un délai de 5 jours ouvrés à réception de votre demande. Vous pouvez également, pour des motifs légitimes, vous opposer au traitement des données vous concernant en adressant votre demande écrite à l'adresse indiquée ci-dessus.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           4.L'accès à certaines sections ainsi que le bon fonctionnement du Service nécessite la transmission de documents ainsi que l'utilisation et le remplissage de formulaires. Ces données, que l'Utilisateur choisit librement de communiquer à Dune sont des éléments d’ordre personnel et confidentiel. Il appartient à l'Utilisateur de prendre toutes les dispositions nécessaires permettant de protéger ses propres données contre toute atteinte.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           5.Les données transmises par l’Utilisateur sont destinées à l’usage exclusif de Dune. Dune garantit la confidentialité des données à caractère personnel recueillies, qui ne peuvent faire l’objet d’aucune cession à des tiers même à titre gracieux.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           6.L’Utilisateur donne son consentement explicite à l'utilisation des données à caractère personnel le concernant, collectées sur l’Application, dans le cadre exclusif du bon fonctionnement des services.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           7.L’Utilisateur est susceptible de recevoir des notifications de la part de Dune via l’application Dune. La réception de ces notifications est librement paramétrable via le menu d’accueil de son terminal mobile.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           4.Limites de responsabilité
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           1.Dans le cadre de sa mission d'information, Dune ne peut être tenu responsable de la non exactitude d'une information. Tout Utilisateur détectant une erreur peut la signaler par courrier électronique à l'adresse Dune@hotmail.fr.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           2.L'utilisateur admet connaître les limitations et contraintes propres au réseau internet et, à ce titre, reconnaît notamment l'impossibilité d'une garantie totale de la sécurisation des échanges de données. Dune ne pourra pas être tenue responsable des préjudices découlant de la transmission de toute information.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           3.Dune ne pourra en aucun cas, dans la limite du droit applicable, être tenue responsable des dommages et/ou préjudices, directs ou indirects, matériels ou immatériels, ou de quelque nature que ce soit, résultant d'une indisponibilité du Service ou de toute utilisation du Service. Le terme « Utilisation » doit être entendu au sens large, c'est-à-dire tout usage de l’application quel qu'il soit, licite ou non.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           4.L'Utilisateur s'engage, d'une manière générale, à respecter l'ensemble de la réglementation en vigueur en France.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           5.Réclamation
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           Pour toute réclamation concernant les informations fournies sur l’Application, Dune met à disposition de tout Utilisateur un formulaire de contact accessible depuis le Site www.Dune.fr conformément aux dispositions de l’article 6-I-5 de la LCEN auprès du service client.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           6.Droit applicable et juridiction compètente
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           1.Le Service se réserve le droit de modifier les termes, conditions et mentions du présent contrat à tout moment.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           2.Le présent contrat est conclu pour une durée indéterminée à compter de l'Utilisation du Service par l'Utilisateur.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           3.Les règles en matière de droit, applicables aux contenus et aux transmissions de données sur et autour de l’application, sont déterminées par la loi française. En cas de litige, n'ayant pu faire l'objet d'un accord à l'amiable, seuls les tribunaux français sont compétents.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginTop": 10,
            }
          }
        >
          •
           7.Propriété intellectuelle
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           1.©Dune, tous droits réservés. La reproduction du contenu de cette application, en tout ou partie, est strictement interdite sans la permission écrite de Dune. Tout autre matériel contenu sur cette application ne nous appartenant pas (photos, textes, images, logos, noms de produits ou de marques…) est la propriété de leurs détenteurs respectifs.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           2.Dune est une marque déposée à l'Institut National de la Propriété Industrielle. Toute reproduction non autorisée de cette marque, logo et signe distinctif constitue une contrefaçon passible de sanctions pénales. Le contrevenant s'expose à des sanctions civiles et pénales rappelées aux articles L. 335.2 et L. 343.1 du code de la Propriété Intellectuelle. Les Services, le Site et l’Application sont protégés par des droits de propriété intellectuelle et/ou autres droits que Dune détient ou dont elle est autorisée à faire usage.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           3.L'Utilisateur ne peut en aucun cas stocker (hors session de connexion, le cas échéant), reproduire, représenter, modifier, transmettre, publier, adapter sur quelque support que ce soit, par quelque moyen que ce soit, ou exploiter de quelque manière que ce soit, les éléments du Service et/ou des Applications sans l’autorisation préalable écrite de Dune.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           4.L'Utilisateur est seul responsable du Contenu Utilisateur qu'il transmet via le Service, ainsi que des textes et/ou opinions qu'il formule dans la « zone commentaire ». Il s'engage notamment à ce que ces données ne soient pas de nature à porter atteinte aux intérêts légitimes de tiers quels qu'ils soient.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           5.Toute extraction et/ou réutilisation sans autorisation d’une ou de plusieurs bases de données tirées ou copiées ou faites à partir du contenu de l’application et/ou des Services, de manière directe ou non, est sanctionnée civilement et pénalement.
        </Text>
        <Text
          style={
            Object {
              "color": "#2A5E85",
              "fontSize": 12,
              "marginBottom": 10,
              "marginLeft": 10,
              "marginTop": 10,
            }
          }
        >
          •
           6.Tout Utilisateur devra respecter toutes les mentions relatives aux droits de propriété intellectuelle figurant aux présentes Conditions Générales d’Utilisation et ne pas les altérer, supprimer, modifier ou autrement y porter atteinte.
        </Text>
      </View>
    </RCTScrollView>
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#999",
            "borderRadius": 5,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "#FFF",
              "fontSize": 14,
            }
          }
        >
          Accepter
        </Text>
      </View>
    </View>
  </View>
</View>
`;
